api/proquest/importer.py

```diff
@@ -728,10 +728,8 @@ def fulfill(
 
     def external_integration(self, db):
         """Return an external integration associated with this object.
-
```

Comment:
Yes, I expected the same and had the same confusion about this (here https://github.com/ThePalaceProject/circulation/pull/37#issuecomment-1029006602)

Let me look into this, now that its confirmed

Reply:
Looks like the merge from main overwrote this (or the conflict resolution did).
Could you point me in the direction of a proquest feed I can test with

Reply:
You can ask Tara in slack for the feed we configure, but the proquest feed is a bit hard to work with because they keep it behind a firewall only accessible to our production servers. If you want to modify the code here as best you can to make sure these changes work with proquest, I will test with proquest feed on server before merging.

Reply:
Changed, but untested